Dockerfile.ct.testing

```diff
@@ -0,0 +1,82 @@
+# MIT License
```

Comment:
why did you make this file? What is different between Dockerfile and Dockerfile.ct.testing?

if its just environment variables, they can be set in the docker compose file

Reply:
Good point. I've deleted Dockerfile.ct.testing and am now using Dockerfile in docker-compose.test.ct.yaml.